test/test_transforms.py

```diff
@@ -2090,5 +1923,193 @@ def test_normalize_3d_tensor():
     torch.testing.assert_close(target, result2)
 
 
+class TestAffine:
+
+    input_img = np.zeros((40, 40, 3), dtype=np.uint8)
+    cnt = [20, 20]
+
+    @pytest.mark.parametrize("pt", [(16, 16), (20, 16), (20, 20)])
+    @pytest.mark.parametrize("i", range(-5, 5))
+    @pytest.mark.parametrize("j", range(-5, 5))
+    def test_make_img_affine(self, pt, i, j):
+        self.input_img[pt[0] + i, pt[1] + j, :] = [255, 155, 55]
```

Comment:
Hm, this almost works 😄
The problem is that this assumes that `test_make_img_affine` is called before the rest of the tests, which isn't an assumption we should make. Also, filling the values of the input_img isn't a test in itself so we should use a different logic here.

This is a good case for using a fixture https://docs.pytest.org/en/6.2.x/fixture.html:

```py
@pytest.fixture(scope='class')
def input_img(self):
    out = np.zeros((40, 40, 3), dtype=np.uint8)
    for pt in ...
        for i in ...
            for j in ...
    ...
    return out
```

and then we can use `input_img` in `PIL_Image` directly

Reply:
Haha yeah it makes sense. Guess I got carried away by pytest's parameterisation :laughing: